CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
```

Comment:
Should we also document the next breaking change for `envVariablePrefix`?

Reply:
It is not a breaking change though.
People can still use `Imperative.envVariablePrefix`, it's just deprecated 😋